README.md

```diff
@@ -6,7 +6,21 @@ This project contains the `moactl` command line tool that simplifies the use of
 
 This guide walks through setting up your first Amazon Red Hat OpenShift cluster using `moactl`.
 
-If you have already [installed the required prerequisites](#Installation-prerequisites), here are the commands you need to create a cluster.
+Before you can start working with `moactl` commands you have to have your AWS privileges set up correctly.
```

Comment:
Before you can start working with `moactl` commands**,** your AWS privileges must be set up correctly.

To set up AWS privileges, run the following command:

Also worth noting that a JIRA card was filed today to rename moactl to "rosa": https://issues.redhat.com/browse/SDA-3130